snakemake_executor_plugin_googlebatch/executor.py

```diff
@@ -110,6 +110,24 @@ def validate(self, job):
                 f"Job {job} has container without image_family batch-cos*."
             )
 
+    def is_preemptible(self, job):
```

Comment:
I tried to update here to use the default provided args - note this wasn't fully tested per a bug, fix is here: https://github.com/snakemake/snakemake/pull/2616